src/services/e-commerce/e-commerce.service.ts

```diff
@@ -1,84 +1,46 @@
-import { TRACK_EVENT } from '../../constants/track-event.constant';
-import { PaqService } from '../paqService/paq.service';
+import { TRACK_EVENT } from '../../constants/track-event.constant'
+import { PaqService } from '../paqService/paq.service'
+import { Product } from '../../interfaces/product'
+import { PaymentInformation } from '../../interfaces/payment'
 
-export function addEcommerceItem(
-  productSKU: string,
-  productName: string,
-  productCategory: string | string[],
-  productPrice: number,
-  productQuantity: number
-) {
-  PaqService.push([
-    TRACK_EVENT.ADD_ECOMMERCE_ITEM,
-    productSKU,
-    productName,
-    productCategory,
-    productPrice,
-    productQuantity,
-  ])
-}
-
-export function removeEcommerceItem(productSKU: string) {
-  PaqService.push([
-    TRACK_EVENT.REMOVE_ECOMMERCE_ITEM,
-    productSKU
-  ])
+export function addEcommerceItem(products: Product[]) {
+  PaqService.push([TRACK_EVENT.ADD_ECOMMERCE_ITEM, products])
 }
 
-export function clearEcommerceCart() {
-  PaqService.push([
-    TRACK_EVENT.CLEAR_ECOMMERCE_CART,
-  ])
+export function removeEcommerceItem(products: Product[]) {
+  PaqService.push([TRACK_EVENT.REMOVE_ECOMMERCE_ITEM, products])
 }
 
 export function getEcommerceItems(): Promise<object> {
```

Comment:
I know that's the old code, but is possible to improve this returned type and make it a more detailed type?